JellyfishClientSdk/Sources/JellyfishClientSdk/JellyfishClientSdk.swift

```diff
@@ -12,6 +12,40 @@ public struct Config {
     }
 }
 
+internal protocol JellyfishWebsocket {
+    var delegate: WebSocketDelegate? { get set }
+    func connect()
+    func disconnect()
+    func write(data: Data)
+}
+
+public class JellyfishClientWebSocket: JellyfishWebsocket {
+  var socket: WebSocket
+  var delegate: WebSocketDelegate? { set{self.socket.delegate = newValue} get{self.socket.delegate}}
```

Comment:
why a getter/setter here if it doesn't do anything than getting and setting a field? can't we just use that field directly?

Reply:
In `JellyfishClientInternal` line 21 the delegate is assigned to our socket but in fact we want to assign it to the real socket  underneath, it was a faster way to write this, I can change it if you want

Reply:
aah ok, let's keep it